lib/any.js

```diff
@@ -683,6 +684,22 @@ module.exports = internals.Any = class {
             }
         }
 
+        if (this._settings) {
+
+            const settings = Hoek.clone(this._settings);
```

Comment:
Good call. This was much messier, so when I pulled `baseType` out of _settings I was happy to have removed a bunch of conditional logic and didn't even see that I could simplify it even more haha.